crates/uv-interpreter/src/discovery.rs

```diff
@@ -219,7 +224,15 @@ fn python_executables<'a>(
             .map(|path| Ok((InterpreterSource::CondaPrefix, path)))
         ).into_iter().flatten()
     )
-    // (4) A discovered environment
+    // (4) The first `python` executable on the `PATH`
+    // N.B. We must filter this interpreter later to ensure it is a virtual environment
+    .chain(
+        sources.contains(InterpreterSource::ShimEnvironment).then(||
+            python_executables_from_search_path(version, implementation).next()
+            .map(|path| Ok((InterpreterSource::ShimEnvironment, path)))
```

Comment:
How do we know this is a shim?

Reply:
We don't. We're going to naively say the first executable in the path _might_ be a shim then verify it during the interpreter query portion.